### PR TITLE
Booking

### PR DESCRIPTION
**Pull request by aryanvichare**

### CODE REVIEW

File: lib/screens/hospital_info.dart [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+  HospitalInfo(
+      {Key key,
+      @required this.name,
+      @required this.image,
+      @required this.cx,
+      this.cy})
+      : super(key: key);
+
+  String mapboxKey =
+      'pk.eyJ1IjoiZWFuZHBpIiwiYSI6ImNra3YwaWxhYjA1cWEybnBmaWEwaXJsdDMifQ.1v9TuJukfLXG9IzY9SufpA';
```
Comment (jessehu312) [inline lib/screens/hospital_info.dart:21]: make sure not private key

File: lib/main.dart [hunk 2]
```diff
@@ -23,21 +24,18 @@ void main() {
       child: MaterialApp(
         title: APP_NAME,
         theme: ThemeData(
-          primarySwatch: Colors.blue,
-          scaffoldBackgroundColor: Colors.white
-        ),
-        initialRoute: '/',
+            primarySwatch: Colors.blue, scaffoldBackgroundColor: Colors.white),
+        initialRoute: '/home',
```
Comment (jessehu312) [inline lib/main.dart:28]: should you move Homepage() into auth.dart? just remove the debug page

File: lib/screens/homepage.dart [hunk 3]
```diff
@@ -1,125 +1,216 @@
...
-      )
-      .toList();
+  List<String> localHospitals = [];
+
+  void fetchLocalPharmacies() async {
+    const LOCATION_API_URL =
+        'https://api.radar.io/v1/search/places?categories=medical-health,pharmacy&near=37.3,-122.03&radius=1000';
+
+    var response = await http.get(LOCATION_API_URL, headers: {
+      'Authorization': 'prj_live_pk_ac938e9fb81e8b90f0670de7074b322c79a6a41b'
```
Comment (jessehu312) [inline lib/screens/homepage.dart:43]: private key? or do we need some sort of .env file?

Comment (aryanvichare) [reply lib/screens/homepage.dart:43]: This is the Radar.io Live Publishable Key. Should be good to use client side

**Pull request merged**